Todo app.js

```diff
@@ -1,88 +1,103 @@
+// User data
+let userList = document.querySelector(".userList");
+let data = [];
+const userData = () => {
+  let userInput = document.getElementById("user-data").value; //Getting the inputed data
+  if (userInput === "") {
+    //this would run if user inputs a empty data
+    alert("Fill in your data");
+    return;
+  }
+  // else if the data isn't empty run this:
+  else {
+    let userOutput = document.createElement("div"); //creted a div
+    userOutput.classList.add("userOutput"); //added a class to the div
+    let un_li = document.createElement("ul"); //created an un-ordered list
+    un_li.style.listStyle = "none"; //Gave style to it
+    let sub_li = document.createElement("li"); //created a list
+    sub_li.textContent = userInput; //Gave it text of whatever the user puts as data
     //created a trash icon
-    let trash=document.createElement('i')
-    trash.classList.add('fa', 'fa-trash-o')
-    trash.setAttribute('aria-hidden',"true")
+    let trash = document.createElement("i");
+    trash.classList.add("fa", "fa-trash-o");
+    trash.setAttribute("aria-hidden", "true");
     // push the elements into a parent div
-    un_li.appendChild(sub_li)
+    un_li.appendChild(sub_li);
     userOutput.appendChild(un_li);
-    userOutput.appendChild(trash)
-    data.unshift(userOutput)
-        data.forEach(data=>{
-            userList.appendChild(data)
-        })
-        //Created a deleted action
-        // add click event listener to userList
-        userList.addEventListener('click', (event) => {
-        // get the clicked element
-        const clickedElement = event.target;
-        
-        // check if the clicked element is a trash icon
-        if (clickedElement.classList.contains('fa-trash-o')) {
-            // get the index of the clicked element's parent div in the data array
-            const index = data.indexOf(clickedElement.parentElement);
-            
-            // remove the clicked element's parent div from the data array
-            if (index > -1) {
-            data.splice(index, 1);
-            }
-            
-            // remove the clicked element's parent div from the DOM
-            clickedElement.parentElement.remove();
+    userOutput.appendChild(trash);
+    data.unshift(userOutput);
+    data.forEach((data) => {
+      userList.appendChild(data);
+    });
+    //Created a deleted action
+    // add click event listener to userList
+    userList.addEventListener("click", (event) => {
+      // get the clicked element
+      const clickedElement = event.target;
+
+      // check if the clicked element is a trash icon
+      if (clickedElement.classList.contains("fa-trash-o")) {
+        // get the index of the clicked element's parent div in the data array
+        const index = data.indexOf(clickedElement.parentElement);
+
+        // remove the clicked element's parent div from the data array
+        if (index > -1) {
+          data.splice(index, 1);

```

Comment:
every other thing is code formatting. Prettier is pretty opionated. sorry about that.